File: pgm/ABC_Java/ABC_456_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final int X = Integer.parseInt (reader.readLine());
        reader.close();
        final String Yes = "Yes";
        final String No = "No";
        final int one = 1;
        final int three = 3;
        final int six = 6;
        System.out.println (one * three <= X && X <= six * three ? Yes : No);
        System.exit (0);
    }
}

File: pgm/ABC_R/ABC_456_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
Yes <- "Yes"
No <- "No"
blank <- ""
one <- 1L
three <- 3L
six <- 6L
X <- arr[one] |>
  as.integer()
if (one * three <= X && X <= six * three) {
  Yes |>
    cat("\n", sep = blank)
} else {
  No |>
    cat("\n", sep = blank)
}
q("no")